### PR TITLE
Fix `res.raw.statusCode` typo

### PR DESCRIPTION
**Pull request by numToStr**

### CODE REVIEW

File: docs/Server.md [hunk 1]
```diff
@@ -183,12 +183,12 @@ custom `onRequest` and `onResponse` hooks.
 
 ```js
 // Examples of hooks to replicate the disabled functionality.
-fastify.addHook('onRequest', (req, reply, done) => {
+fastify.addHook('onRequest', (req, res, done) => {
```
[the remark this answers is not in the document]
Comment (numToStr) [reply docs/Server.md:186]: yeah sure.

[the remark this answers is not in the document]
Comment (numToStr) [reply docs/Server.md:186]: Just want to point out that `reply`, `res`, and `request`, `req` are used interchangeably throughout the documentation. Maybe It needs to be consistent.

[the remark this answers is not in the document]
Comment (jsumners) [reply docs/Server.md:186]: > Just want to point out that `reply`, `res`, and `request`, `req` are used interchangeably throughout the documentation. Maybe It needs to be consistent.

Our documentation needs a significant overhaul in all respects.

[the remark this answers is not in the document]
Comment (numToStr) [reply docs/Server.md]: Thanks for letting me know.

**Comment by mcollina** [comment]
Good spot!

**Pull request merged**